mlir/test/Dialect/Vector/vector-warp-distribute.mlir

```diff
@@ -494,6 +494,62 @@ func.func @warp_scf_for_multiple_yield(%arg0: index, %arg1: memref<?xf32>, %arg2
 
 // -----
 
+// CHECK-PROP-LABEL:   func @warp_scf_for_multi_reduce(
+//   CHECK-PROP-NOT:   vector.warp_execute_on_lane_0
+//       CHECK-PROP:   scf.for {{.*}} -> (vector<1x4xf32>) {        
+//       CHECK-PROP:     scf.for {{.*}} -> (vector<1x4xf32>) {
+//       CHECK-PROP:       vector.transfer_read {{.*}} : memref<2x32x40x384xf32>, vector<1x4xf32> 
```

Comment:
Can we check the transfer_read index? That's an important part of the distribution--how we are deseralizing the 1-D processor ID there.

Reply:
Acutally ignore this--we distribute transfer read in another code path.. Would be good to unify.